src/status_im/contexts/preview/quo/inputs/address_input.cljs

```diff
@@ -25,8 +25,8 @@
         :blur?                 (:blur? @state)
         :show-blur-background? true}
        [quo/address-input
-        (merge @state
-               {:on-scan       #(js/alert "Not implemented yet")
+        (merge (dissoc @state :scanned-value)
```

Comment:
This preview has to be adapted to the changes of the component

Reply:
it's adapted, or could you elaborate please

Reply:
There's no `:scanned-value` anymore and it should be replaced with `:default-value` here and in the descriptors, no? Also, the preview descriptor for the scanned/default-value seems redundant, as it no longer changes the input.

Reply:
scanned-value is still needed in preview to mock scanner result

Reply:
I see now :+1: